stable/aws-load-balancer-controller/values.yaml

```diff
@@ -169,3 +169,10 @@ defaultTags: {}
 
 podDisruptionBudget: {}
 #  maxUnavailable: 1
+
+serviceMonitor:
```

Comment:
Each defined property in this file should be documented. Please refer to https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml

Reply:
done. nested attributes have been explained in the README